tests/functional/pyocf/types/cache.py

```diff
@@ -478,16 +517,26 @@ def generate_attach_config(
 
         return attach_cfg
 
+    def generate_activate_config(self, device, open_cores=True, perform_test=False):
+        device_cfg = Cache.generate_device_config(device, perform_test=perform_test)
 
-    def attach_device(
-        self, device, force=False, perform_test=False, cache_line_size=None, open_cores=True
+        activate_cfg = CacheStandbyActivateConfig(
+            _device=device_cfg, _open_cores=open_cores,
+        )
+
+        return activate_cfg
+
+    def _attach_device(
+        self,
+        device,
+        standby=False,
+        force=False,
+        perform_test=False,
+        cache_line_size=None,
+        open_cores=True,
     ):
         attach_cfg = self.generate_attach_config(
-            device,
-            force,
-            perform_test,
-            cache_line_size,
-            open_cores=open_cores
+            device, force, perform_test, cache_line_size, open_cores=open_cores
```

Comment:
Do we need this helper function? To me this attach logic seems to be smaller than amount of boilerplate generated when trying to avoid duplicating this logic.

Reply:
removed